src/lib.rs

```diff
@@ -941,6 +941,23 @@ fn test_is_on_curve_var() {
     assert!(AffinePoint::identity().is_on_curve_vartime());
 }
 
+#[test]
+fn test_affine_point_generator() {
+    AffinePoint::generator();
+}
+
+#[test]
+fn test_affine_point_generator_is_on_curve() {
+    AffinePoint::generator().is_prime_order();
+}
+
+#[test]
+fn test_affine_point_generator_is_not_identity() {
+    assert_ne!(
+        ExtendedPoint::from(AffinePoint::generator().mul_by_cofactor()),
+        ExtendedPoint::identity()
+    );
+}
```

Comment:
I don't actually think that this is something that adds value as testing. I know the generator is not the identity, it's not something that has to be tested.

When I said add tests I was refering to check that the generator point that the function provides has indeed order P and not 8P 4P 8 4 etc..

Reply:
I think stating it isn't the identity is important. Either way, a check that the order is prime, has been done in a test. However, I will rename this function.

Reply:
The function `test_affine_point_generator_has_order_p`, checks this and passes.